### PR TITLE
Support AZURE_ENVIRONMENT_NAME for Azure Gov

### PR DESCRIPTION
**Pull request by kim0**
Need to persist this env var `AZURE_ENVIRONMENT_NAME` for Azure gov to work. It helps the azure sdk select the correct endpoints.

### CODE REVIEW

File: postgres-appliance/scripts/configure_spilo.py [hunk 1]
```diff
@@ -757,7 +757,7 @@ def write_wale_environment(placeholders, prefix, overwrite):
                 'WALE_S3_ENDPOINT', 'AWS_ENDPOINT', 'AWS_REGION', 'AWS_INSTANCE_PROFILE',
                 'WALG_S3_SSE_KMS_ID', 'WALG_S3_SSE', 'WALG_DISABLE_S3_SSE', 'AWS_S3_FORCE_PATH_STYLE']
     azure_names = ['WALG_AZ_PREFIX', 'AZURE_STORAGE_ACCOUNT', 'AZURE_STORAGE_ACCESS_KEY',
-                   'AZURE_STORAGE_SAS_TOKEN', 'WALG_AZURE_BUFFER_SIZE', 'WALG_AZURE_MAX_BUFFERS']
+                   'AZURE_STORAGE_SAS_TOKEN', 'WALG_AZURE_BUFFER_SIZE', 'WALG_AZURE_MAX_BUFFERS', 'AZURE_ENVIRONMENT_NAME']
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/configure_spilo.py:760]: https://github.com/zalando/spilo/pull/688/checks#step:7:6:
```
postgres-appliance/scripts/configure_spilo.py:760:121: E501 line too long (123 > 120 characters)
```

Please also update the documentation

Comment (kim0) [reply postgres-appliance/scripts/configure_spilo.py]: @CyberDem0n Done, please check

**Comment by FxKu** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Comment by CyberDem0n** [comment]
Merged, thank you @kim0!

**Pull request merged**